### DISCUSSION TITLE
How to get all the guilds a user is the owner of?

### DISCUSSION THREAD

**Discussion by NyanFroggy** [Rapptz/discord.py#6781] [Q&A]
I want to get the list of servers I am the owner of.
Is there a function like this-
`guilds=client.get_guild(id).owner.ownedguilds()`

**Comment by AbstractUmbra** [comment]
This does not exist, no.
You could achieve this would a simple iteration though?

```py
for guild in bot.guilds:
  if guild.owner_id == <your ID here>:
    print(f"I own {guild.name}")
```

This is of course a very barebones and pseudocode concept. But the idea is there.

**Reply by NyanFroggy** [reply]
ok, thanks
